package.json

```diff
@@ -73,6 +73,7 @@
     "node-emoji": "^1.10.0",
     "shelljs": "^0.8.3",
     "spektate": "^0.1.5",
+    "ts-node": "^8.4.1",
```

Comment:
May not need this

Reply:
I will take a look since it was added from debugging in VS Code.

Reply:
Move to devDependencies please